sewer/auth_providers/tests/test_utils.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+
+class MockResponse(object):
+    """
+    mock python-requests Response object
+    """
+
+    def __init__(self, status_code=200, content=None):
+        if not content:
+            content = {}
+
+        try:
+            content.update(
+                {
+                    "something": "ok",
+                    "result": [{"name": "example.com", "id": "some-mock-dns-zone-id"}],
+                }
+            )
+        except AttributeError:
+            json_loads = json.loads
+        else:
+            json_loads = lambda a: a
+
+        self.content = json.dumps(content).encode()
+        self.text = json_loads(self.content.decode())
+
+        self.headers = {}
+        self.status_code = status_code
+
+    def json(self):
+        return json.loads(self.content.decode())
+
+
+class mockLibcloudDriverZone(object):
+    """
+    A mock of a dns zone in a libcloud drivers dns
+    """
+
+    id = "mock-zone-id-1"
+
+    def create_record(self, name, type, data):
+        pass
+
+
+class mockLibcloudDriver(object):
+    """
+    a mock of libcloud.dns.drivers.auroradns.AuroraDNSDriver class
+    """
+
+    def __init__(self, key, secret):
+        pass
+
+    def get_zone(self, domainSuffix):
+        mock_zone = mockLibcloudDriverZone()
+        return mock_zone
+
+    def list_records(self, zone):
+        import collections
+
+        DnsRecords = collections.namedtuple("DnsRecords", "id name type")
+        one_dns_record = DnsRecords(id="1", name="_acme-challenge", type="TXT")
+        records = [one_dns_record]
+        return records
+
+    def get_record(self, zone_id, record_id):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/komuW/sewer/pullRequest?prid=5264928)